SwiftCompilerSources/Sources/ExperimentalRegex/CMakeLists.txt

```diff
@@ -1,21 +0,0 @@
-# This source file is part of the Swift.org open source project
-#
-# Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
-# Licensed under Apache License v2.0 with Runtime Library Exception
-#
-# See http://swift.org/LICENSE.txt for license information
-# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
-
-file(GLOB_RECURSE _LIBSWIFT_EXPERIMENTAL_REGEX_SOURCES
-  "${EXPERIMENTAL_STRING_PROCESSING_SOURCE_DIR}/Sources/_MatchingEngine/*.swift")
-set(LIBSWIFT_EXPERIMENTAL_REGEX_SOURCES)
-foreach(source ${_LIBSWIFT_EXPERIMENTAL_REGEX_SOURCES})
-  file(TO_CMAKE_PATH "${source}" source)
-  list(APPEND LIBSWIFT_EXPERIMENTAL_REGEX_SOURCES ${source})
-endforeach()
-message(STATUS "Using Experimental String Processing library for libswift ExperimentalRegex (${EXPERIMENTAL_STRING_PROCESSING_SOURCE_DIR}).")
-
-add_swift_compiler_module(ExperimentalRegex
-  "${LIBSWIFT_EXPERIMENTAL_REGEX_SOURCES}"
-  Regex.swift)
-
```

Comment:
Did you accidentally deleted this and the `Regex.swift` source?

Reply:
Good catch! Forgot to git add.